Codeinwp/optimole-wp
# Add initial crawl on connect

### All Submissions:

 

### Changes proposed in this Pull Request:
This crawls images when you connect to Optimole service. More details here: https://github.com/Codeinwp/optimole-lambda-functions/pull/388
 
 

Closes https://github.com/Codeinwp/optimole-service/issues/643.

### How to test the changes in this Pull Request:
- Create a new website and add some images to it, you can add 100-150 images for testing. (If you don't want to upload, it's easy to use https://wordpress.org/plugins/instant-images/)
- Once you connect your website to Optimole and the loading stops, you should see some images crawled and ready for display in the dashboard.
- Make sure you're testing on a new instance, not a website which has already been crawled by Optimole.
- It's an old video but the result should be something like this: https://www.loom.com/share/7a32c468673b4b9884cb7cf51c71355d

### Other information:
